app/js/components/LabResultEntry.jsx

```diff
@@ -94,7 +95,11 @@ export class LabResultEntry extends PureComponent {
   renderForm(selectedLabConcept) {
     const {
       CONSTANTS, conceptMembers, selectedPatient, patients, history: { location: { state } },
+      isDidNotPerformCheckboxSelected, dispatch, formId,
     } = this.props;
+    if (!(isDidNotPerformCheckboxSelected)) {
+      dispatch(change(formId, `obs|path=did-not-perform-dropdown|conceptPath=${CONSTANTS.labResultsDidNotPerformReasonQuestion}`, ''));
```

Comment:
If `did Not Perform checkbox` is not checked, I delete/clear the value of the did not perform reason in the redux store.

Reply:
I created a util method to create the selector for the obs, so if we make a change to the format again (like when I changed "concept" to "conceptPath") we don't need to update all our references. We should switch to use this:

https://github.com/openmrs/openmrs-react-components/blob/master/src/features/form/util.js

Sorry I forgot to point this out when I created it.